### PR TITLE
Added variables for stock location

### PR DESCRIPTION
**Pull request by SergeoLacruz**

### CODE REVIEW

File: docs/report/context_variables.md
Comment (matmair) [inline docs/report/context_variables.md:159]: Question: Isn't this a bit redundant? The parent location is already one level up

File: docs/report/context_variables.md
Comment (matmair) [inline docs/report/context_variables.md:162]: Suggestion
```suggestion
| structural | True if the location is structural | 
```

**Review by matmair — APPROVED** [review]
Thank you for your contribution! I have some suggestions regarding the phrasing.

**Comment by SergeoLacruz** [comment]
Done :-)

**Review by matmair — APPROVED** [review]

**Comment by matmair** [comment]
Thanks again, we will merge once the maintainer is back online in a few days.

**Comment by SchrodingersGat** [comment]
@SergeoLacruz many thanks for the contributions :)

**Pull request merged**
